docs/docsite/rst/presentations/openstack-modules-intro.rst

```diff
@@ -0,0 +1,394 @@
+==================================
+Intro to Ansible OpenStack Modules
+==================================
+
+This document contains a presentation in `presentty`_ format. If you want to
+walk through it like a presentation, install `presentty` and run:
+
+.. code:: bash
+
+    presentty docs/docsite/rst/presentations/openstack-modules-intro.rst
+
+The content is hopefully helpful even if it's not being narrated, so it's being
+included in the `shade` docs.
+
+.. _presentty: https://pypi.python.org/pypi/presentty
+
+Ansible: Community, Collaboration, and the OpenStack Modules you know and love
```

Comment:
Do we want to delete/move content  from here to `Ansible: Community, Collaboration, and the OpenStack Modules you know and love` given it brakes `reveal.js`?